packages/devtools_app/lib/src/flutter/screen.dart

```diff
@@ -76,6 +76,28 @@ abstract class Screen {
   }
 }
 
+/// Defines a page shown in the DevTools TabBar if [conditionalLibrary] is
+/// present in the connected application.
+abstract class ConditionalScreen extends Screen {
+  const ConditionalScreen(
+    this.conditionalLibrary, {
+    String title,
+    IconData icon,
+    Key tabKey,
+  }) : super(
+          DevToolsScreenType.conditional,
+          title: title,
+          icon: icon,
+          tabKey: tabKey,
+        );
+
+  /// Library name on which we determine whether to include this screen in
```

Comment:
nit: on which we determine -->  that determines

Reply:
done